tests/appsec/test_traces.py

```diff
@@ -334,3 +334,54 @@ class Test_DistributedTraceInfo:
 
     def test_main(self):
         assert False, "Test not implemented"
+
+
+@rfc("https://docs.google.com/document/d/1xf-s6PtSr6heZxmO_QLUtcFzY_X_rT94lRXNq6-Ghws/edit?pli=1")
+@features.security_events_metadata
+class Test_ExternalWafRequestsIdentification:
+    def setup_external_wafs_header_collection(self):
+        self.r = weblog.get(
+            "/headers",
+            headers={
+                "X-Amzn-Trace-Id": "Root=1-65ae48bc-04fb551979979b6c57973027",
+                "CloudFront-Viewer-Ja3-Fingerprint": "e7d705a3286e19ea42f587b344ee6865",
+                "Cf-Ray": "230b030023ae2822-SJC",
+                "X-Cloud-Trace-Context": "105445aa7843bc8bf206b12000100000/1",
+                "X-Appgw-Trace-id": "ac882cd65a2712a0fe1289ec2bb6aee7",
+                "X-SigSci-RequestID": "55c24b96ca84c02201000001",
+                "X-SigSci-Tags": "SQLI, XSS",
+                "Akamai-User-Risk": "uuid=913c4545-757b-4d8d-859d-e1361a828361;status=0",
+            },
+        )
+
+    @missing_feature(library="golang", reason="Not implemented yet")
+    @missing_feature(library="java", reason="Not implemented yet")
+    @missing_feature(library="dotnet", reason="Not implemented yet")
+    @missing_feature(library="python", reason="Not implemented yet")
+    @missing_feature(library="ruby", reason="Not implemented yet")
+    @missing_feature(library="php", reason="Not implemented yet")
+    @missing_feature(library="nodejs", reason="Not supported yet")
```

Comment:
I think we could remove these decorators, because we are using the manifests and there is only one test method on this class, right?

Reply:
Yes, I am not very familiar with how tests are conducted/tracked yet